packages/fxa-settings/src/styles/tailwind.scss

```diff
@@ -37,3 +38,14 @@ body {
     @apply transition duration-150;
   }
 }
+/* stylelint-disable */
+@responsive {
+  .drop-down-menu {
+    @apply w-64 bg-white absolute shadow-md rounded;
```

Comment:
The Monitor menu is definitely more rounded (maybe `rounded-lg`) but I'd say match whatever Figma is looking like on the avatar menu and we can check with design later.

Reply:
updated to rounded-lg, I think it looks okay on both of them now